.circleci/main-config.yml

```diff
@@ -1079,35 +1095,15 @@ parameters:
 workflows:
   version: 2
 
-  # Run all tests on DSS project
-  dss-build-test-workflow:
+  # Triggered from a local script to test PR changes
```

Comment:
Idea of this whole PR is that the github status check can be setup so that a single workflow passes.
So the separate workflows we had to run tests for DSS, DSM and API docs have been combined into this one workflow.
It is unfortunate that  CircleCi does not allow conditional logic to run jobs that are part of a workflow conditionally, so we will run all the jobs, but, as you saw in previous changes, each job will do as little as possible.